frontend/src/components/common/RadioImageSelectGroup.tsx

```diff
@@ -38,7 +39,7 @@ const RadioImageSelectButton = (props: any) => {
       cursor: "pointer",
       borderWidth: "1px",
       borderRadius: "6",
-      borderColor: "hubbard.100",
+      borderColor: "#D8DDE0",
```

Comment:
can you add this colour as a variant in the `theme.tsx`? Seeing it's used three times